code/data_setup/parse_alov_bb.sh

```diff
@@ -1,19 +0,0 @@
-i=0
-while read line
-do
-    files[ $i ]="$line"        
-    (( i++ ))
-done < <(find $1 -type f)
-
-
-echo ${#files[@]}
-echo ${files[0]} | cut -d / -f 3 
-
-for file in "${files[@]}"
-do 
-    sub_dir="$(echo $file | cut -d / -f 3 | cut -d . -f 1),"
```

Comment:
The `cut` stuff first split the file path on the `/`, grabbing the 3rd part of that split (`-f 3`), and then did it again on `.` and grabbed the first part. The comma was so that I could add it into the `.csv` as it was being generated. I thought that was the easiest place to put a comma to get into `.csv` format as another column.